src/slicevec.rs

```diff
@@ -0,0 +1,1025 @@
+#![allow(unused_variables)]
+#![allow(missing_docs)]
+
+use super::*;
+
+/// A slice-backed vector-like data structure.
+///
+/// This is a very similar concept to `SliceVec`, but instead
```

Comment:
typo?

```suggestion
/// This is a very similar concept to `ArrayVec`, but instead
```

Reply:
Yeah, the first draft of this file was just duplicating the old file and running some regex XD